kumascript/macros/PWASidebar.ejs

```diff
@@ -7,18 +7,31 @@ const sidebar = {
       name: "/docs/Web/Progressive_web_apps/Guides",
       contents: [
         "/docs/Web/Progressive_web_apps/Guides/What_is_a_progressive_web_app",
+        "/docs/Web/Progressive_web_apps/Guides/Installing",
         "/docs/Web/Progressive_web_apps/Guides/Making_PWAs_installable",
         "/docs/Web/Progressive_web_apps/Guides/Offline_and_background_operation",
+        "/docs/Web/Progressive_web_apps/Guides/Caching",
         "/docs/Web/Progressive_web_apps/Guides/Best_practices",
       ],
     },
     {
       name: "/docs/Web/Progressive_web_apps/Tutorials",
       contents: [
+        {
+          name: "/docs/Web/Progressive_web_apps/Tutorials/Intro",
```

Comment:
Question: can `name` be something that's not a link, or is there a way to avoid having the same text ("PWA tutorial: Introduction") in the top-level item like:

* PWA tutorial: Introduction
  * PWA tutorial: Introduction
  * PWA tutorial: Base HTML and CSS
  * ...

It might be nice to have
  
* PWA tutorial: CycleTracker application
  * PWA tutorial: Introduction

We also have `PWA tutorial:` before each sub-item, but I guess this is something for `short-title` in content, right?

![image](https://github.com/mdn/yari/assets/43580235/383b6168-9e34-4c11-9c5f-c8a60d2c51ac)

Reply:
And not related to these changes, but while I'm looking at the sidebar, I think "Reference" could be improved as "PWA reference" for the short-title

Reply:
I see it looks intentional to leave out PWA from "Reference" because we have "Guides", "Tutorials", and "How to" so it would mean prepending PWA there, also or having some inconsistency.

Reply:
Anyway: we could of course let sidebar authors specify a different name. But I'd prefer to avoid this if possible, because then we have to have more authored content in sidebar specifications, and also the localizations of them, and all this makes them harder to maintain. I'm much happier having the sidebar pull titles from content.

Reply:
>I'm much happier having the sidebar pull titles from content.
yes, that's perfect. Thanks, Will 👍🏻

Reply:
>This is a pretty common pattern in MDN sidebars. I don't like it either, but I don't think inventing a different title is an ideal solution either. Ideally IMO we would find a way to let the top level page here be a link, then we wouldn't need to repeat the item.

We're maybe looking for a "label" for the list items that categorizes what the items are. I always think of this as a good hint for how this might look in future https://docusaurus.io/docs/sidebar/items#sidebar-item-category